test/functional/ui/output_spec.lua

```diff
@@ -176,16 +176,13 @@ describe("shell command :!", function()
       command(helpers.iswin()
         and [[nnoremap <silent>\l :!dir /b bang_filter_spec<cr>]]
         or  [[nnoremap <silent>\l :!ls bang_filter_spec<cr>]])
-      local result = (helpers.iswin()
-        and [[:!dir /b bang_filter_spec]]
-        or  [[:!ls bang_filter_spec    ]])
       feed([[\l]])
       screen:expect([[
                                                              |
         {1:~                                                    }|
         {1:~                                                    }|
         {4:                                                     }|
-        ]]..result..[[                            |
+                                                             |
```

Comment:
is this right? here we go from one to zero visible echos.

Reply:
I don't know why it cannot be captured in tests.

But in current code, double echoed.

Reply:
Well if we fix some places but break others, it is not an improvement.